### ISSUE TITLE
Running `rubocop --auto-gen-config` ignores `RSpec/BeforeAfterAll` offense

### ISSUE THREAD

**Issue by mwolman** [rubocop/rubocop#10827]
## Expected behavior

After running `rubocop --auto-gen-config`, all the offenses are added to the `.rubocop_todo.yml` file, and then, if you run `rubocop -a` you should have 0 offenses detected.

## Actual behavior

I ran `rubocop -a` and there's still 1 offense related to the `RSpec/BeforeAfterAll` cop 

## Steps to reproduce the problem

1. In your `rails_helper` file, define something like this:
```ruby
RSpec.configure do |config|
  
  config.before(:all) do
    ...
  end

  config.use_transactional_fixtures = true

end
```

2. Install `rubocop`, `rubocop-rails` and `rubocop-rspec` with the versions mentioned below
3. Run `rubocop --auto-gen-config`
4. Run `rubocop -a`
5. Check that you have 1 offense

## RuboCop version

```
1.7.0 (using Parser 3.1.2.0, rubocop-ast 1.18.0, running on ruby 2.7.6 arm64-darwin21)
  - rubocop-performance 1.5.2
  - rubocop-rails 2.11.3
  - rubocop-rspec 2.4.0
```
